addons/website_sale/views/templates.xml

```diff
@@ -197,14 +197,11 @@
                 <div class="container py-2 oe_website_sale">
                     <div class="products_pager form-inline justify-content-center">
                         <t t-call="website_sale.search">
-                            <t t-set="_classes">o_website_sale_search</t>
+                            <t t-set="_classes">o_website_sale_search col-12 col-md-8</t>
                         </t>
                         <t t-call="website_sale.pricelist_list">
                             <t t-set="_classes">ml-2</t>
                         </t>
-                        <t t-call="website.pager">
-                            <t t-set="_classes">ml-2</t>
-                        </t>
```

Comment:
You did not like the pager ?

Reply:
There is still a pager below the listed products. Plus, what is the point to let this pager next to the new autocomplete input ? Does this make sense ?

Reply:
I don't see how this is related to your task. You improve the search, it does not mean clients will not want to browse all products or search results... so yes there is still a need for a pager unless I am missing something. And I think it is nice to have one on top and one below, especially since it does not take more space.